docs/openapi.yaml

```diff
@@ -0,0 +1,399 @@
+openapi: 3.0.0
+info:
+  title: Signalen Informatievoorziening Amsterdam (SIA) API spec V1"
+  description: One of the tasks of the municipality of Amsterdam is to maintain 
+    public spaces like roads, parks. When citizens have complaints about public
+    spaces they can leave these complaints with the municipality. SIA, the
+    Signalen Informatievoorziening Amsterdam, receives these complaints and is
+    used to track progress on their resolution. SIA provides an API that is used
+    both by the SIA frontend and external systems that integrate with SIA.
+  contact:
+    email: "datapunt@amsterdam.nl"
+  version: "TBD"
+servers:
+  - url: "https://acc.api.data.amsterdam.nl"
+    description: "Datapunt acceptatie server for SIA testing and acceptation."
+paths:
+  /signals/v1/private/signals/:
+    get:
+      description: Signals list endpoint
+      parameters:
+        - name: "created_before"
+          in: query
+          description: Filter returns only signals created before specified
+            datetime.
+          schema:
+            type: string
+            format: date-time
+          required: false
+        - name: "created_after"
+          in: query
+          description: Filter returns only signals created after specified
+            datetime.
+          schema:
+            type: string
+            format: date-time
+          required: false
+        - name: "updated_before"
```

Comment:
Tried out the format for created_before, created_after, updated_before, updated_after. Should be of the form 2019-02-01 13:00:00.